python/foglamp/services/common/microservice.py

```diff
@@ -67,7 +77,8 @@ def __init__(self, default_config):
             })
             self._core_microservice_management_client.create_configuration_category(config_payload)
             config = self._core_microservice_management_client.get_configuration_category(category_name=category)
-            host = config['management_host']['value']
+            is_local_services = True if config['local_services']['value'].lower() in ['true', 't', '1', 1] else False
```

Comment:
` if config['local_services']['value'].lower() == 'true'`, remove all other, it only creates confusion and we can add redundancy everywhere !

Reply:
All other values are perfectly valid for type "boolean", hence required.